content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -189,7 +189,7 @@ and pre-created PVs, but you'll need to look at the documentation for a specific
 to see its supported topology keys and examples.
 
 {{< note >}}
-   If you choose to use `waitForFirstConsumer`, do not use `nodeName` in the Pod spec
+   If you choose to use `WaitForFirstConsumer`, do not use `nodeName` in the Pod spec
```

Comment:
That is still incorrect. The correct spelling is "WaitingForFirstConsumer" according to the API spec.